config/environments/production.rb

```diff
@@ -150,7 +150,7 @@
   require_relative '../../lib/rack_throttle/rules'
 
   rules = [
-    { method: "POST", limit: ENV['GRAPH_API_RATE_LIMIT'] },
+    { method: "POST", limit: ENV['GRAPH_API_RATE_LIMIT'] || 1 },
```

Comment:
Why in this PR? How this is connected with Keycloak ?

Reply:
Not having a default value for that limit was preventing the creation of the students through the API. It was causing any `POST` call to the API to be denied with a `503 Rate limit exceeded` . Is this a bad change? Should we rely on just the value of the env var and let `POSTs` be blocked if it's not defined?

Reply:
No. I was just curious how this affects keycloak integration. It's good to have defaults not crash whole app like if some VAPID keys are not defined ;)